experimental/IntersectionTheory/src/Main.jl

```diff
@@ -1295,7 +1295,7 @@ function schur_functor(F::AbstractBundle, λ::Partition)
   λ = conjugate(λ)
   X = F.parent
   w = _wedge(sum(λ), chern_character(F))
-  S, ei = polynomial_ring(QQ, "e#" => 1:length(w))
+  S, ei = polynomial_ring(QQ, ["e$i" for i in 1:length(w)])
```

Comment:
```suggestion
  S, ei = polynomial_ring(QQ, "e#" => 1:length(w))
```
this produces the same, but is more efficient